CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### New features
 
+* [#7998](https://github.com/rubocop-hq/rubocop/pull/7998): Updates Rake to version 13. ([@RicardoTrindade][])
```

Comment:
This changelog entry is unnecessary. Gemfile is a development dependency, not a runtime dependency.